File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> numsSet = new HashSet();
        int max = 0;
        for(int i = 0;i< nums.length;i++)
            numsSet.add(nums[i]);
        for(int i=0;i<nums.length;i++){
            if(!numsSet.contains(nums[i]-1)){ //it is the beginning
                int count = 1;
                int current = nums[i];
                while(true){
                    if(!numsSet.contains(current+1))
                        break;
                    current+=1;
                    count+=1;
                }
                if(max<count) max=count;
            }
        }
        return max;
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result=[0]*2
        i,j = 0, len(nums) -1
        while i !=j:
            if nums[i]+nums[j] == target:
                result[0]=i+1
                result[1]=j+1
                break
            elif nums[i]+nums[j] < target:
                i+=1
            else:
                j-=1
        return result
